src/main/scala/iog/psg/cardano/experimental/cli/model/MetadataJsonFile.scala

```diff
@@ -24,18 +36,27 @@ object MetadataJson {
 
   def asString(metadataJson: MetadataJson): String = metadataJson.asJson.noSpaces
 
+  implicit val stringSeqEncoder: Encoder[Seq[String]] = (seq: Seq[String]) =>
+    if (seq.size == 1) Encoder.encodeString(seq.head) else Encoder.encodeSeq[String].apply(seq)
+
+  implicit val nftFileEncoder: Encoder[NftFile] = deriveEncoder
+
   implicit val encoder: Encoder[MetadataJson] = {
 
     case NftMetadataJson(policyId: PolicyId, nfts) =>
       Json.obj(("721",
       Json.obj(
         (policyId.value, Json.obj(nfts.map(nft =>
-          (nft.label, Json.obj(
+          (nft.assetName, Json.obj(
```

Comment:
[CIP-25](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0025) currently has 2 versions. In version 2 policyId and assetName should be raw bytes. Should we change it, so it uses version 2 or we stay on version 1?

Reply:
Is there an option to support both and determine which version we should use base on some property in JSON or if we fail to deserialize it in a certain way?

Reply:
If there is an easy fallback than yes, if not, we should pick the one that Dish is using now

Reply:
To support both for creating nfts, we need to take a parameter which specifies the version. Maybe it will be better if we default to the one Dish is using for now. @mcsherrylabs do you know which version is Dish using now?